### PR TITLE
[13.0] [ADD] cmis_mail: store mail composer attachments in CMIS

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**

### CODE REVIEW

File: cmis_mail/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<section class="s_text_block">
```
Comment (lmignon) [inline cmis_mail/static/description/index.html:1]: The description doesn't describe the current addon :smirk:

**Review by lmignon — CHANGES_REQUESTED** [review]
@ThomasBinsfeld Thank you for this new addon. A small comment otherwise LGTM...

File: cmis_mail/static/description/index.html (hunk 1 above)
Comment (ThomasBinsfeld) [reply cmis_mail/static/description/index.html:1]: Now it does :ok_hand:  Thanks for the review.

**Comment by sbidoul** [comment]
Nice module!

**Pull request closed without merging**